rpcs3/Emu/Cell/Modules/cellSubDisplay.cpp

```diff
@@ -29,7 +29,7 @@ void fmt_class_string<CellSubDisplayError>::format(std::string& out, u64 arg)
 error_code cellSubDisplayInit(vm::ptr<CellSubDisplayParam> pParam, vm::ptr<CellSubDisplayHandler> func, vm::ptr<void> userdata, u32 container)
 {
 	cellSubDisplay.todo("cellSubDisplayInit(pParam=*0x%x, func=*0x%x, userdata=*0x%x, container=0x%x)", pParam, func, userdata, container);
-	return CELL_OK;
+	return CELL_SUBDISPLAY_ERROR_ZERO_REGISTERED;
```

Comment:
can you put a description here as to why not returning CELL_OK here?

Reply:
// tells the cellSubDisplay that there is not a vita or psp remote play device enabled

I think just from looking at CELL_SUBDISPLAY_ERROR_ZERO_REGISTERED documentation I could find from the ps3devwiki (which is not much)